docs/DeveloperGuide.adoc

```diff
@@ -165,15 +166,41 @@ when the list data are changed.
 Mark. Each `Mark`, in turn, comprises a `UniqueBookmarkList`, a `ReminderAssociation` object, an
 `AutotagController`, and a `FolderStructure`.
 
+.Structure of the Bookmark package
```

Comment:
```suggestion
.Structure of the Bookmark Package
```

capitalize like all the other captions?

Reply:
actually nvm i looked at all the captions and theyre not consistent anyway, we can just fix it all later

Reply:
i think we should all switch to sentence case, since some captions quite long

Reply:
Mm okay. Will do that later.